BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -796,27 +798,17 @@ To address these concerns, TLS implementations SHOULD NOT use static DH keys and
 The following considerations and recommendations represent the current state of the art regarding certificate revocation, even though no complete and efficient solution exists for the problem of checking the revocation status of common public key certificates {{RFC5280}}:
 
 
-* Although Certificate Revocation Lists (CRLs) are the most widely supported mechanism for distributing revocation information, they have known scaling challenges that limit their usefulness (despite workarounds such as partitioned CRLs and delta CRLs).
-
+* Certificate revocation is an important tool when recovering from attacks on the TLS implementation, as well as cases of misissued certificates. TLS implementations MUST implement a strategy to distrust revoked certificates.
+* Although Certificate Revocation Lists (CRLs) are the most widely supported mechanism for distributing revocation information, they have known scaling challenges that limit their usefulness, despite workarounds such as partitioned CRLs and delta CRLs.
 * Proprietary mechanisms that embed revocation lists in the Web browser's configuration database cannot scale beyond a small number of the most heavily used Web servers.
```

Comment:
I'm not sure this is completely accurate. Mozilla has been experimenting with CRLite for a while and it scales better than what we say here. See https://blog.mozilla.org/security/2020/01/09/crlite-part-1-all-web-pki-revocations-compressed/

Reply:
> I'm not sure this is completely accurate. Mozilla has been experimenting with CRLite for a while and it scales better than what we say here. See https://blog.mozilla.org/security/2020/01/09/crlite-part-1-all-web-pki-revocations-compressed/

Two years later, the CRLite mode in my Firefox is still 1 (i.e., measure, do not enforce). So I'm a bit skeptical.

So maybe:

> Although Certificate Revocation Lists (CRLs) are the most widely supported mechanism for distributing revocation information, they have known scaling challenges that limit their usefulness, despite workarounds such as partitioned CRLs and delta CRLs. The more modern [CRLite] and the follow-on [Let's Revoke] build on the availability of Certificate Transparency [CT] logs and aggressive compression to allow practical use of the CRL infrastructure, but at the time of writing, neither solution is deployed for client-side revocation processing at scale.

Reply:
LGTM